pkgs/tools/admin/credhub-cli/default.nix

```diff
@@ -11,6 +11,10 @@ buildGoModule rec {
     sha256 = "1j0i0b79ph2i52cj0qln8wvp6gwhl73akkn026h27vvmlw9sndc2";
   };
 
+  patches = [
+    ./test.diff
```

Comment:
Please use fetchpatch with https://github.com/cloudfoundry-incubator/credhub-cli/commit/4bd1accd513dc5e163e155c4b428878ca0bcedbc.patch.

Reply:
I'm not sure that's safe since the PR hasn't (and may never) get merged into cloudfoundry-incubator - but *shrug*, done.